Code/app/src/main/java/com/cmpt362/fitcheck/firebase/AuthenticationUtil.kt

```diff
@@ -1,4 +1,4 @@
-package com.cmpt362.fitcheck.authentication
+package com.cmpt362.fitcheck.firebase
```

Comment:
This authenticationUtil isn't used at all in the firebase.kt so I feel like it should be in the authentication package no?